guides/common/modules/proc_adding-a-subnet.adoc

```diff
@@ -4,6 +4,11 @@
 You must add information for each of your subnets to {ProjectServer} because {Project} configures interfaces for new hosts.
 To configure interfaces, {ProjectServer} must have all the information about the network that connects these interfaces.
 
+[IMPORTANT]
+====
+In an IPv6 network with unmanaged DHCP and DNS, do not assign DHCP {SmartProxy} and Reverse DNS {SmartProxy} to your subnet.
```

Comment:
This is IMHO a bug that the user can select this in the first place. Is there work tracked to make that impossible?

Reply:
I wouldn't mark it as a bug yet. We may start supporting DHCPv6 management in the future. The same goes for DNS, maybe we will be able to support managed DNS for IPv6 networks later on (for example in retroactive mode, where we update the DNS once we get the facts from the host).

Reply:
Until we develop the feature, we should make it impossible set unsupported options. Because of that I'd qualify it as a bug.

Reply:
Let's create an issue on Redmine then. Sounds fine by me.

Reply:
It looks like this thread will be resolved by an issue on the code side. I'll resolve this thread then (but it would be nice if you could add a link to the Redmine issue for reference).

Reply:
Unresolving because reverse DNS should work. There is explicit code for it and I'd like to see some reproducer that shows it's broken before documenting it. Then we can, while we fix the bug, document a known issue.

Reply:
Interesting, but I'm having trouble wrapping my head around why this is an issue. Is there a bug report with the reproducer steps? Is it something like:

* User creates a new host
* User doesn't select a compute resource which can provide an IPv6 address
* User adds a NIC
* User selects IPv6 subnet with reverse DNS set
* User doesn't set an IPv6 address

Now the user is presented with an error, but how is that a problem? When is selecting an IPv6 subnet without setting an IPv6 address valid?

If you are provisioning with IPv6 with unmanaged DHCPv6 then I'd assume you don't set the IPv6 subnet and instead rely on it to be set later when it's set via facts: https://github.com/theforeman/foreman/blob/2a81d51f7793f82bc5687952789e716b278083ff/app/models/host/base.rb#L512-L517